components/progress-bar/index.jsx

```diff
@@ -0,0 +1,122 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
+/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */
+
+// Implements the [Progress Bar design pattern](https://lightningdesignsystem.com/components/progress-ring/) in React.
+// Based on SLDS v2.4.5
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+// ### shortid
+// [npmjs.com/package/shortid](https://www.npmjs.com/package/shortid)
+// shortid is a short, non-sequential, url-friendly, unique id generator
+import shortid from 'shortid';
+import { PROGRESS_BAR } from '../../utilities/constants';
+
+
+const propTypes = {
+	/**
+	 * HTML id for component.
+	 */
+	id: PropTypes.string,
+	/**
+	 * CSS classes to be added to tag with `.slds-progress-bar`. Uses `classNames` [API](https://github.com/JedWatson/classnames).
+	 */
+	className: PropTypes.oneOfType([
+		PropTypes.array,
+		PropTypes.object,
+		PropTypes.string,
+	]),
+	/**
+	 * Label for the progress bar
+	 */
+	label: PropTypes.string,
```

Comment:
Please allow node

Reply:
@interactivellama  I have updated it into `node` type